src/ImageProcessorCore/Formats/Bmp/README.md

```diff
@@ -1,8 +1,8 @@
-﻿Encoder/Decoder adapted from:
-
-https://github.com/yufeih/Nine.Imaging/
-https://imagetools.codeplex.com/
-
-TODO:
-
-- Add support for all bitmap formats.
+﻿Encoder/Decoder adapted from:

```

Comment:
I wonder... Did the encoding change for this file or do you have tabs instead of spaces in VS?